client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -167,8 +169,15 @@ const PluginsBrowserListElement = ( props ) => {
 	);
 };
 
-const InstalledInOrPricing = ( { sitesWithPlugin, isWpcomPreinstalled } ) => {
+const InstalledInOrPricing = ( {
+	sitesWithPlugin,
+	isWpcomPreinstalled,
+	plugin,
+	period = 'monthly',
+} ) => {
 	const translate = useTranslate();
+	const priceSlug = `${ plugin?.slug?.replaceAll( '-', '_' ) }_${ period }`;
```

Comment:
The paid plugins data include a `variations` object. `variations[period].product_slug` should return the `priceSlug`. 

```
variations: {
monthly: {
product_id: 903,
product_slug: "woocommerce_subscriptions_monthly",
},
yearly: {
product_id: 902,
product_slug: "woocommerce_subscriptions_yearly",
},
},
```

Reply:
This is the marketplace products response https://public-api.wordpress.com/wpcom/v2/marketplace/products

Reply:
From the marketplace products response, we get all plugin data. Therefore we have the `variations` to be passed as an argument to `getProductDisplayCost`

Reply:
Found it in the API response, however some keys get filtered out when normalizing the plugin data: https://github.com/Automattic/wp-calypso/blob/f25dcede8e20d96890f9b9bcfefb8d7d58f30399/client/lib/plugins/utils.js#L64-L99

I'm currently looking for a clean way to pass the variations key as well.

Reply:
This filtering is only used on WPorg and WPcom API reponses. As the WPorg API response has no `variations` key, I think it's safe to add it to this list. 👍